canvas/fizz-21-py/.gitignore

```diff
@@ -0,0 +1,145 @@
+# Originated from https://github.com/github/gitignore/blob/master/Python.gitignore
```

Comment:
Can you go ahead and load most of this into the root `.gitignore` file? Except for `.vscode/`, because generally we *do* want to share these settings across users.